packages/vscode-extension/src/commonlib/telemetry.ts

```diff
@@ -93,11 +94,30 @@ export class VSCodeTelemetryReporter extends vscode.Disposable implements Teleme
     }
 
     if (
-      properties[TelemetryProperty.ProjectId] === "unknown" ||
-      properties[TelemetryProperty.ProjectId] === undefined
+      !properties[TelemetryProperty.ProjectId] ||
```

Comment:
It seems overriding a common property is supported, right? If so, is it better to separate these conditions because if the users of this function want to override a common property they have to specify **all three properties**?

Reply:
Offline discussed with @KennethBWSong, it is not intended behavior to override common properties.